docs/developer-docs/integrations/sns/testing/testing-locally.md

```diff
@@ -30,106 +30,95 @@ These scripts have been most tested with a stack that looks like this:
 
 ## Testing via `sns-testing` repo
 
-A common way to test an SNS locally is to use the `sns-testing` [repo](https://github.com/dfinity/sns-testing#readme).
-
-### Examples of what it tests
-
-The `sns-testing` repo can help with testing the lifecycle of launching an SNS, by running the commands listed [here](https://github.com/dfinity/sns-testing#sns-lifecycle).
-
-For simplicity, you can see the stages of decentralization and how they line up with the CLI commands. 
-
 :::info
 To use the `sns-testing` commands and their arguments properly, please consult the `sns-testing` [README](https://github.com/dfinity/sns-testing#sns-lifecycle).
 :::
 
-### Testing stages of decentralization
 
-Below, we match the stages of decentralization to the commands that the `sns-testing` repo has. Some developers have dapps that do not match the narrow cases of `sns-testing` so the table also includes a column for what other developers have experienced.
+### Testing stages of the SNS launch process
+
+For simplicity, we next map the stages introduced in the [SNS launch stages documentation](../launching/launch-summary-1proposal.md) and in the documentation [Commands & actions to go through SNS launch](../launching/launch-steps-1proposal.md) to the relevant scripts in the `sns-testing` repo so you can learn what part of the launch is tested in which script.
+Note that some developers have dapps that do not match the narrow cases of `sns-testing` so the table also includes a column for what other developers have experienced.
 
 <table border="1">
     <tr>
         <th>Stage Number</th>
         <th>Stage</th>
-        <th>Example `sns-testing` command</th>
+        <th>Example in `sns-testing`</th>
         <th>For other developers</th>
     </tr>
     <tr>
         <td>0</td>
         <td>Developers deploy a dapp to the Internet Computer</td>
-        <td><code>./deploy_test_canister.sh</code></td>
+        <td><code> TODO ./deploy_test_canister.sh</code></td>
         <td>Custom scripts used to deploy dapps (e.g. multi-canister dapps, use nix, etc...)</td>
     </tr>
     <tr>
         <td>1</td>
         <td>Dapp developers choose the initial parameters of the SNS for a dapp</td>
-        <td rowspan="4"><code>./deploy_sns.sh</code></td>
+        <td rowspan="2"><code>TODO ./deploy_sns.sh</code></td>
     </tr>
     <tr>
         <td>2</td>
-        <td>Dapp developers submit NNS proposal so they can deploy to the SNS subnet</td>
+        <td>Submit NNS proposal to create SNS</td>
     </tr>
     <tr>
         <td>3</td>
-        <td>Proposal #1 (of 3) is passed or rejected</td>
+        <td>Dapp developers add NNS root as co-controller of dapp</td>
+        <td><code>TODO</code></td>
+        <td><code></code></td>
     </tr>
     <tr>
         <td>4</td>
-        <td>Dapp developers trigger the SNS canisters to be created on SNS subnet</td>
+        <td>The NNS proposal is decided</td>
+        <td rowspan="7"><code>TODO ./deploy_sns.sh</code></td>
```

Comment:
The way I would represent this is with stages 4-8 should have no script and it should be "automated". There should be a step in between 8 and 9, which is `participate in the swap` and the code link is `./participate_in_sns_swap.sh`. then the current step 9 should also have no code link and be "automated"

Reply:
I now added a step as you suggested but I did not add a number.
This way, the layout here would match what you said but still the "Stages" in the Launch consdier "what devs do / what the system does" and also it is a bit less error prone to not change numbering now.
let me know if you disagree with that.

Reply:
🙏